Core/Src/CAN/CAN_handler.c

```diff
@@ -26,13 +30,13 @@ void initialize_CAN(FDCAN_HandleTypeDef* hfdcan,
         Error_Handler(); // NOLINT
     }
 
-    // funcao para registrar a funcao de callback
+    // function to register the custom can receive callback routine
```

Comment:
```suggestion
    // Function to register the custom CAN receive callback
```
Repete isso aqui em baixo tbm.
E para *todos* os comentários que vc mexeu:
Colocar letra maiuscula em todos os comentários (código todo ta sem padrão disso e sei que ta feio rs, mas vamos tentar deixar isso melhor)
Tenta simplificar e reduzir os comentarios e tirar os textos que da pra induzir pelo código.